src/hotspot/os/posix/signals_posix.cpp

```diff
@@ -1246,7 +1241,8 @@ void set_signal_handler(int sig, bool do_check = true) {
 
   // Save handler setup for later checking
```

Comment:
JDK-8283337 changed this comment to: // Save handler setup for possible later checking